product_listings_manager/tests/test_rest_api_v1.py

```diff
@@ -4,22 +4,25 @@
 
 from product_listings_manager.app import app
 
-# Flask < 1.0 does not have get_json.
-from flask import Response
-import json
-if not hasattr(Response, 'get_json'):
-    def get_json(self, force=False, silent=False, cache=True):
-        """ Minimal implementation we can use this in the tests. """
-        return json.loads(self.data)
-    Response.get_json = get_json
-
 
 @pytest.yield_fixture
 def client():
     client = app.test_client()
     yield client
 
 
+class TestRoot(object):
```

Comment:
If call this TestRoot, it should be Api root, maybe TestApiRoot?

Reply:
Good point. I've changed this to `TestIndex`, to match the new `Index` class in `rest_api_v1.py`